front/app/containers/Admin/reporting/components/ReportBuilderPage/EmptyState.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import EmptyState from './EmptyState';
+import { render, screen } from 'utils/testUtils/rtl';
+import * as FeatureFlag from 'hooks/useFeatureFlag';
+
+const mockFeatureFlag = FeatureFlag as { default: () => boolean };
+
+describe('EmptyState', () => {
+  it('renders with an enabled button to create a report when the feature flag is on', () => {
```

Comment:
```suggestion
  it('renders with an enabled button to create a report when the feature is allowed by the pricing plan', () => {
```

We're only testing allowed in EmptyState.tsx, so let's make that clear?